examples/storybook/src/stories/index.ts

```diff
@@ -0,0 +1,27 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import * as algoliasearch from 'algoliasearch/lite';
+
+import { NgAisModule } from 'angular-instantsearch';
+import '!style-loader!css-loader!../styles.css';
+
+import { wrapWithHits } from '../wrap-with-hits';
+
+const WithSearchClient = wrapWithHits({
+  template: '',
+  searchClient: algoliasearch(
+    'latency',
+    '6be0576ff61c053d5f9a3225e2a90f76'
+  ),
+});
+const stories = storiesOf('InstantSearchJS', module).addDecorator(
+  moduleMetadata({
+    imports: [NgAisModule.forRoot()],
+    declarations: [WithSearchClient],
+    providers: [],
+  })
+);
+stories
+  .add('with algoliasearch search client', () => ({
+    component: WithSearchClient
+  }));
+
```

Comment:
The list of examples is minimale for now
![screenshot at nov 05 11-16-20](https://user-images.githubusercontent.com/2982512/47992172-7be04880-e0ec-11e8-98b4-15a600ffd055.png)